int32 predicted_class
string species_name
float64 confidence
